.github/workflows/android-device-test.yml

```diff
@@ -83,6 +77,33 @@ jobs:
         with:
           files: "${{ matrix.test.testFile }}, ${{ matrix.test.appFile }}"
 
+      - name: Upload external data for benchmark
+        if: matrix.test.name == 'Android Benchmark'
+        run: |
```

Comment:
I would recommend putting this in a script in the repo and referencing this script instead of writing it all in the CI code, this way you can also test it locally.

Reply:
Like this?

Reply:
Yup